### ISSUE TITLE
Make the bulding process outputs three pip packages

### ISSUE THREAD

**Issue by wangkuiyi** [sql-machine-learning/elasticdl#1982]
Currently, our `.travis.yml` defines a building process that the generated `.whl` file is in a Docker image.  However, following the idea in https://github.com/sql-machine-learning/elasticdl/issues/1981, we are going to release three .whl files, instead of a Docker image.  This change requires us to change the building process to generate .whl files on the host computer, other than in a Docker image.

**Comment by skydoorkai** [comment]
We can build python codes on the host computer with `setup.py` without any problem.
The only issue we need to solve is  PS Go codes in elasticdl building. The host computer needs to install Golang.
We can add it to the pre-request.

**Comment by workingloong** [comment]
We can check in the pre-compiled binary files of PS in the repo. In the building `.whl` process, we can directly use the binary files.

**Comment by wangkuiyi** [comment]
> We can check in the pre-compiled binary files of PS in the repo. In the building `.whl` process, we can directly use the binary files.

No. The basic rule of using a versioning system is NOT TO CHECKIN BINARIES.

Hopefully, we can refer to https://github.com/elasticdl/recordio/blob/develop/python/setup.py for an example of how to bundle Go binaries in wheel files.
